pkg/handler3/handler.go

```diff
@@ -192,7 +249,14 @@ func (o *OpenAPIService) HandleGroupVersion(w http.ResponseWriter, r *http.Reque
 			if err != nil {
 				return
 			}
-			w.Header().Set("Etag", etag)
+			w.Header().Set("Etag", strconv.Quote(etag))
```

Comment:
Why does the ETag HTTP header need to be quoted? Might be worth a `//` comment.

Reply:
Added a reference to the MDN page although I'm not sure how much value that provides since it is a standard.